ceph-salt-formula/salt/ceph-salt/update/update.sls

```diff
@@ -0,0 +1,53 @@
+{% import 'macros.yml' as macros %}
+
+{{ macros.begin_stage('Update all packages') }}
+
+install required packages:
+  pkg.installed:
+    - pkgs:
+      - lsof
+    - failhard: True
+
+update packages:
```

Comment:
After the package update, we need to do:

```
# in case any package update re-writes sudoers /etc/sudoers.d/<ssh_user>
{{ macros.sudoers('configure sudoers after package update') }}
```
See https://github.com/ceph/ceph-salt/pull/318

Reply:
Done